src/api/CustomAction/types.ts

```diff
@@ -61,7 +61,7 @@ export interface ParsedCustomAction {
 }
 
 export enum CustomActionRight {
-  OWNER = 'owner',
+  MANAGE = 'manage',
```

Comment:
`write` seems more idiomatic to me, especially in conjunction with `read`.

It's also more accurate in my opinion as:
* if I'm a query manager, I've got `write` access on all the non-private queries.
* if I'm the owner of a query, I've got `write` access on that query, but I don't have the right to `manage` other queries.

```suggestion
  WRITE = 'write',
```

Reply:
ping @EricMallard as this is also applicable to https://github.com/Linkurious/linkurious-rest-client/pull/686

Reply:
@gmasclet @EricMallard 
I already answered in the slack discussion :)

Reply:
As agreed on slack, let's stick with 'manage'. I close this comment.